### ISSUE TITLE
Project Library: search and filter doesn't show up after server update&restart. requires user to refresh

### ISSUE THREAD

**Issue by hirokiterashima** [WISE-Community/WISE-Portal--Deprecated--#113]
When we update the server with the latest codebase and restart it, 

the search and filter side panel in the Project Library is hidden. 

I've seen this problem before and we got a user report this week after our server update and restart on Monday.  I don't remember if there is a js error.

Something to keep an eye out for.

**Comment by breity** [comment]
Hmm, I can't replicate on either production or locally.  Let me know if you
or someone else sees it again and under what conditions.

On Thu, Mar 21, 2013 at 3:05 PM, Hiroki Terashima
notifications@github.comwrote:

> When we update the server with the latest codebase and restart it,
> 
> the search and filter side panel in the Project Library is hidden.
> 
> I've seen this problem before and we got a user report this week after our
> server update and restart on Monday. I don't remember if there is a js
> error.
> 
> Something to keep an eye out for.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/WISE-Community/WISE-Portal/issues/113
> .

## 

Jonathan Lim-Breitbart
Feature Developer and Lead Designer
Web-based Integrated Science Environment (WISE)
http://wise.berkeley.edu

**Comment by hirokiterashima** [comment]
yes, let's keep an eye on it next time we update the server, which is either Fri or Mon afternoon.
